2nd-gen/uppercase/README.md

```diff
@@ -0,0 +1,30 @@
+# Firebase SDK for Cloud Functions 2nd Gen Quickstart - Realtime Database
+
+This quickstart demonstrates using **Firebase SDK for Cloud Functions** setup with a **Firebase database**.
+
+
+## Introduction
+
+This sample app does two things:
+ - Create messages in the Firebase Realtime Database using a simple HTTPS request which is handled by a v2 HTTPS Firebase Function. Writing to the Realtime Database is done using the Firebase Admin SDK. 
+ - When a message gets added in the Realtime Database, a v2 Firebase Function triggers and automatically makes these messages all uppercase.
+
```

Comment:
Check with egilmorez... "...a v2 function triggers and...."

Reply:
👍